lib/data_update_scripts/20200526181850_update_public_reaction_counts_again.rb

```diff
@@ -0,0 +1,10 @@
+module DataUpdateScripts
+  class UpdatePublicReactionCountsAgain
+    def run
+      ActiveRecord::Base.connection.execute("UPDATE articles SET public_reactions_count = positive_reactions_count, previous_public_reactions_count = previous_positive_reactions_count WHERE id < #{Article.count / 2}")
+      ActiveRecord::Base.connection.execute("UPDATE articles SET public_reactions_count = positive_reactions_count, previous_public_reactions_count = previous_positive_reactions_count WHERE id > #{Article.count / 2}")
+      ActiveRecord::Base.connection.execute("UPDATE comments SET public_reactions_count = positive_reactions_count WHERE id < #{Comment.count / 2}")
+      ActiveRecord::Base.connection.execute("UPDATE comments SET public_reactions_count = positive_reactions_count WHERE id > #{Comment.count / 2}")
```

Comment:
I ran this in production console earlier, and don't think I need to adjust our database timeout for this.

Reply:
Can you comment out the first data update script so we don't end up running this twice?

Also, since this will be the final script we should have a `>=` for each one bc an ID could be equal to the count divided by 2.